frameworks/helloworld/tests/test_region_awareness.py

```diff
@@ -81,15 +81,8 @@ def test_region_config_update_does_not_succeed(local_service):
     sdk_plan.wait_for_completed_deployment(config.SERVICE_NAME)
     change_region_config(REMOTE_REGION)
     plan = sdk_plan.get_deployment_plan(config.SERVICE_NAME)
-    assert plan.get('errors', [])
-
-    sdk_cmd.svc_cli(config.PACKAGE_NAME, config.SERVICE_NAME, 'pod replace hello-0')
-    sdk_plan.wait_for_in_progress_recovery(config.SERVICE_NAME)
-    sdk_plan.wait_for_completed_recovery(config.SERVICE_NAME)
 
-    pod_region = get_pod_region(config.SERVICE_NAME, 'hello-0')
-
-    assert pod_region == LOCAL_REGION
+    assert plan.get('errors', [])
```

Comment:
I would really consider making a check for errors explicit. What about:
```python
assert "Cannot change region from FOO to BAR" in plan.get("errors", [])
```
Whenever I read the line as it is, my first thought is always that we are checking that there are no errors.

Reply:
I think `some_dict.get(key, default_empty_value)` is about as idiomatic Python as you'll ever see.